src/realm-settings/RealmSettingsSection.css

```
@@ -176,3 +176,12 @@ article.pf-c-card.pf-m-flat.kc-login-settings-template
 .kc-row-drag-button {
   padding: var(--pf-global--spacer--sm);
 }
+
+.kc-help-text {
+  margin: 
+    var(--pf-global--spacer--sm)
+    var(--pf-global--spacer--sm)
+    var(--pf-global--spacer--sm)
+    var(--pf-global--spacer--sm);
```

If the margin is the same on all sides I believe you can provide only a single value and get the same result.
```suggestion
  margin: var(--pf-global--spacer--sm);
```